src/openloco/windows/stationwnd.cpp

```diff
@@ -718,6 +721,61 @@ namespace openloco::ui::windows::station
         }
     }
 
+    // 0x00491BC6
+    static void sub_491BC6()
+    {
+        auto posId = 0;
+        for (coord_t y = 0; y < 0x3000; y += 0x20)
+        {
+            for (coord_t x = 0; x < 0x3000; x += 0x20)
+            {
+                if (_byte_F00484[posId] & (1 << 0))
+                {
+                    map_pos pos = { x, y };
+                    tilemgr::map_invalidate_tile_full(pos);
+                }
+                posId++;
+            }
+        }
+    }
+
+    // 0x00491D70
+    static void sub_491D70(openloco::station* station, uint16_t dx)
```

Comment:
`setStationCatchmentDisplay` or something similar. Its basically calculating what tiles need to be displayed as in the catchment.